docs/docs/configuration/taskdef.md

```diff
@@ -21,10 +21,12 @@ Conductor maintains a registry of worker tasks.  A task MUST be registered befor
   "timeoutPolicy": "TIME_OUT_WF",
   "retryLogic": "FIXED",
   "retryDelaySeconds": 600,
-  "responseTimeoutSeconds": 3600,
```

Comment:
`"responseTimeoutSeconds": 3600` gives a validation error. It should be `1200` or less.

`ownerEmail` seems to be mandatory now. Relates to https://github.com/Netflix/conductor/issues/2542